### PR TITLE
Add instructions for getting tech notifications in Slack

### PR DESCRIPTION
**Pull request by staceysalamon-aiven**
# What changed, and why it matters
Update article on getting notifications for services and add instructions on how to get the email notifications in Slack.

### CODE REVIEW

**Comment by staceysalamon-aiven** [comment]
@victr This is ready for you to review/merge now.

**Review by victr — APPROVED** [review]

**Review by victr — APPROVED** [review]

**Comment by victr** [comment]
thanks, lgtm 👍 but looks like my approval is not enough, probably don't have write access

**Comment by staceysalamon-aiven** [comment]
> thanks, lgtm 👍 but looks like my approval is not enough, probably don't have write access

Thank you! I will just wait for Cloudflare to finish the build and merge it when ready.

**Review by wojcik-dorota — APPROVED** [review]

**Pull request merged**
